rust/sdk-processor/src/steps/common/parquet_extractor_helper.rs

```diff
@@ -0,0 +1,20 @@
+use crate::parquet_processors::{ParquetTypeEnum, ParquetTypeStructs};
+use std::collections::HashMap;
+
+/// Fill the map with data if the table is opted in for backfill-purpose
+pub fn add_to_map_if_opted_in_for_backfill(
+    opt_in_tables: &Option<Vec<String>>,
+    map: &mut HashMap<ParquetTypeEnum, ParquetTypeStructs>,
+    enum_type: ParquetTypeEnum,
+    data: ParquetTypeStructs,
+) {
+    if let Some(ref backfill_table) = opt_in_tables {
+        let table_name = enum_type.to_string();
+        if backfill_table.contains(&table_name) {
```

Comment:
Why don't we use bitflags for the opt-in tables (similar to the deprecated tables)? This might not be very efficient, especially since this will be called very frequently.

Reply:
good callout! I will explore an option of reusing the bigflags we have as well